recipes/lmdb/all/CMakeLists.txt

```
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+# LMDB library:
+
+add_library(lmdb
+    source_subfolder/lmdb.h
+    source_subfolder/mdb.c
+    source_subfolder/midl.c
+    source_subfolder/midl.h
+)
+
+set_target_properties(lmdb PROPERTIES PUBLIC_HEADER
+    source_subfolder/lmdb.h
+)
+
+if(NOT WIN32)
+    target_link_libraries(lmdb PUBLIC pthread)
+endif()
+
+if(WIN32 AND BUILD_SHARED_LIBS)
+  set_target_properties(maxminddb PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS ON)
+endif()
```

This property can be set unconditionally (or `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` defined always).
This property is only applied on Windows + shared libraries.
```suggestion
set_target_properties(maxminddb PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

---

Good point. Thanks!